backup/backup_to_dropbox.py

```diff
@@ -9,10 +9,18 @@
 file_name = sys.argv[1]
 
 # Initliaze a Dropbox client
-access_token = os.environ['DROPBOX_ACCESS_TOKEN']
-client = dropbox.Dropbox(access_token)
-
-with open(file_name, 'rb') as f:
+app_key = os.environ["DROPBOX_APP_KEY"]
```

Comment:
Can we use `os.getenv` here for consistency?

Reply:
Not resolved

Reply:
Removed env dependency in `get_dropbox_tokens.py`

Reply:
hmm. okay.